### PR TITLE
refactor(protocol): slightly change defender monitors

### PR DESCRIPTION
**Pull request by dantaik**

### CODE REVIEW

**Review by davidtaikocha — APPROVED** [review]

**Review by k-kaddal — APPROVED** [review]

**Comment by dantaik** [comment]
@k-kaddal could you own this PR to make the deployment work? I've add the secrets to our github repo.

File: packages/protocol/monitors/.env.example [hunk 1]
```diff
@@ -0,0 +1,2 @@
+DEFENDER_API_SECRET=
```
Comment (dantaik) [inline packages/protocol/monitors/.env.example:1]: I have added these secrets to github.

**Comment by dantaik** [comment]
I think it will be a good idea to move the monitors to a new package: `packages/monitors`, then put files in `packages/monitors/defender/mainnet`

**Comment by dantaik** [comment]
Ready to be reviewed?

**Pull request merged**